requirements/locked.yaml

```diff
@@ -0,0 +1,26 @@
+PyYAML: '6.0'
```

Comment:
who will update these?
See the [Notion doc ](https://www.notion.so/gridai/Addressing-CI-Robustness-in-Lightning-30ef643f11b245ebb12205c55779e1c3) for a suggestion how to keep them updated.

Reply:
in this PR it is manual, but in the following PR we can add workflow weekly generating new/actual/latest versions and there is change, it will create PR to our repo...